helper/install_devel_packages.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+function os_id {
+    grep -E "^ID=" /etc/os-release | cut -f2 -d= | tr -d \"
```

Comment:
Why not just do `source /etc/os-release` and use the `$ID` variable?

Reply:
yes would be easy but is it expected to be sourced ? I see e.g things like ANSI_COLOR that contains escape sequences. If some funny guy just adds code it gets executed on source. Maybe that's a bit paranoid but a simple grep avoids any source magic especially on files outside my control. So it's a matter of trust :)

Reply:
Also keep in mind the documentation tells the user to call ```install_devel_packages``` with root permissions. This is required to install software but it would also mean we source /etc/os-release as root. I think that should be prevented. Thoughts ?

Reply:
The file is designed to be sourced, yes.

Reply:
ok, I changed it, let's hope this will never be abused